lib/modules/platform/bitbucket-server/index.ts

```diff
@@ -373,11 +374,13 @@ export async function getBranchPr(branchName: string): Promise<BbsPr | null> {
 }
 
 // istanbul ignore next
-export async function refreshPr(number: number): Promise<void> {
+export async function refreshPr({
+  number: prNo,
```

Comment:
Is this renaming necessary? I think either keep it as-is, or change the type itself

Reply:
This was me trying to keep consistency with `updatePr()`, I'll iron it out on the next pass 👍